docs/notifications.md

```diff
@@ -0,0 +1,35 @@
+# Notification Development Instructions
+
+The purpose of this document is to provide ZAP developers with simple instructions on how to add a notification.
+
+## ADDING a Notification:
+
+In order to add a notification, call the setNotification function in zap/src-electron/query-notification.js. setNotification takes the following arguments:
+
+- db => database object (object)
+- type => the type of notification you wish to define (string)
```

Comment:
is there a list of existing types?

Reply:
Not yet but I could add a couple to this page to make it work or delete it. I would prefer to keep it as it is a nice identifier for the user. For example, there could be a complicated error message that a user can quickly identify as a specific type of problem.

Reply:
It would sort of be like the subject to an email